src/bootstrap/bootstrap.py

```diff
@@ -315,7 +315,7 @@ def build_triple(self):
         try:
             ostype = subprocess.check_output(['uname', '-s']).strip().decode(default_encoding)
             cputype = subprocess.check_output(['uname', '-m']).strip().decode(default_encoding)
-        except (subprocess.CalledProcessError, WindowsError):
+        except subprocess.CalledProcessError:
```

Comment:
Ah yeah so this clause was added in ad88d50fb specifically for Windows, so it'd be great to at least have the Windows behavior here stay the same (although I'm not sure how to do that best)